HHS/Head-Start-TTADP
# [TTAHUB-275] configure aws elastic search

## Description of change

This is the first pass of AWS Elasticsearch configuration for the BE.

## How to test

- Review code
- Run all unit tests
- Make sure docker config makes sense and properly starts up

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-275


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
